st2actions/st2actions/query/mistral/v2.py

```diff
@@ -115,9 +113,13 @@ def _get_workflow_result(self, exec_id):
         """
         execution = executions.ExecutionManager(self._client).get(exec_id)
 
-        output = jsonify.try_loads(execution.output) if execution.state in DONE_STATES else None
+        result = jsonify.try_loads(execution.output) if execution.state in DONE_STATES else {}
+        result['workflow'] = {
```

Comment:
I'm trying not to break compatibility. Right now, the WF output parameters are directly under the WF result.  I don't want to move those output parameters to another nested level.  I also don't want to have name conflicts either so I move the WF state and state_info under result['workflow'].  How about if we do something like

result = {
    'output1': 'value1',
    'output2': 'value2',
    ...
    'extra': {
        'state': 'SUCCESS',
        'state_info': None,
        'tasks': [
            {
                'name': 'task1',
                'state': 'SUCCESS',
                'state_info': None,
                ...
            },
            ...
        ]
    }
}